src/platform/ESP32/bluedroid/BLEManagerImpl.cpp

```diff
@@ -610,7 +1117,7 @@ CHIP_ERROR BLEManagerImpl::InitESPBleLayer(void)
     }
 
     // Set the maximum supported MTU size.
-    err = MapBLEError(esp_ble_gatt_set_local_mtu(ESP_GATT_MAX_MTU_SIZE));
+    err = MapBLEError(esp_ble_gatt_set_local_mtu(256));
```

Comment:
So, if i understand, this hardcoding will always set initial MTU to 256. Shouldn't we support the max instead ?

Reply:
and the application needs to use 256 size I think the reason behind it would be :
In matter specs it is mentioned that 
-> Value of the maximum receive window size supported by the server, specified in units of BTP packets where each packet may be up to 244 bytes in length. This maximum was chosen so a single BTP segment can fit into a single 255 byte BLE link layer PDU